components/base-node.js

```diff
@@ -44,6 +46,18 @@ module.exports = {
    },
 
   /**
+   * Standard component error handling - send a message to the error port and throw Error
+   *
+   * @param message message to be sent to error port
+   * @param errorPort handle to the component's error port
+   */
+   handleError: function( message, errorPort ) {
```

Comment:
I think we're going to want to move to a model of logging errors, but I'm okay with keeping this as is for the moment, at least until states are implemented.